unit_tests/test_config.py

```diff
@@ -178,13 +178,13 @@ def test_10_mananger_regression():
     def test_12_scylla_version_ami(self):
         os.environ.pop('SCT_AMI_ID_DB_SCYLLA', None)
         os.environ['SCT_CLUSTER_BACKEND'] = 'aws'
-        os.environ['SCT_SCYLLA_VERSION'] = '4.6.3'
+        os.environ['SCT_SCYLLA_VERSION'] = '5.1.8'
```

Comment:
can you move all the those integration test fixes to it's own commit ? i.e. those are not related to gemini docker image

Reply:
done